app/api/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::CommentsController < ApplicationController
```

Comment:
- [ ] It would be better if the README file contains all the information needed to run the app locally